src/providers/ldap/sdap_async_connection.c

```diff
@@ -1274,8 +1274,15 @@ static void sdap_kinit_done(struct tevent_req *subreq)
 
     }
 
-    DEBUG(SSSDBG_CONF_SETTINGS,
-          "Could not get TGT: %d [%s]\n", result, sss_strerror(result));
+    if (result == EFAULT || result == EIO || result == EPERM) {
```

Comment:
Again - what is the reason for splitting this into two different versions based on `result` value?